### PR TITLE
add categories for new problemcodes

### PR DESCRIPTION
**Pull request by bransonf**
This PR updates the categorization of problemcodes. This may have to become a regular thing, as the City's API is still relatively poor at categorizing requests.

### CODE REVIEW

**Comment by bransonf** [comment]
AppVeyor is failing due to this:

```
* installing *source* package 'stlcsb' ...
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : package 'tidyselect' was installed by an R version with different internals; it needs to be reinstalled for use with this R version
ERROR: lazy loading failed for package 'stlcsb'
* removing 'C:/projects/stlcsb/stlcsb.Rcheck/stlcsb'
In R CMD INSTALL

```

File: inst/extdata/admin.csv [hunk 1]
```diff
@@ -75,4 +75,7 @@ Tent Encampment
 TEST FOR BETTY
 Vacant Unit Appeal
 Vacant Unit Appeal t
-VBS Inquiry
\ No newline at end of file
+VBS Inquiry
+Misc-City Counselor
+Loop Trolley
+Test-Do not use
```
Comment (walker) [inline inst/extdata/admin.csv:81]: @bransonf Added in error. This won't show up on next update.

**Comment by walker** [comment]
This data set will have a "GROUP" column in the CSV by next Monday. It'll appear thereafter.

There is not a guaranteed 1-to-1 match between GROUP in the CSV version of this data and GROUP Index in the Open311 API version of the same data due to differences in the way the data is generated.

**Comment by bransonf** [comment]
Thanks again, @walker 
I'll be revisiting CSB categorization entirely sometime next week.

@chris-prener Closing this PR in anticipation of an improved one next week...

**Pull request closed without merging**